### PR TITLE
Add speed bumps to bug filing

### PR DESCRIPTION
**Pull request by jsoref**
### Short description
@Habbie is suffering from people not reading the open source items in the current issue template. This adds some check boxes...

### Checklist

I have:
- [x] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [ ] compiled this code
- [ ] tested this code
- [ ] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added or modified regression test(s)
- [ ] added or modified unit test(s)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -9,8 +9,8 @@ assignees: ''
 
 <!-- Hi! Thanks for filing an issue. It will be read with care by human beings. Can we ask you to please fill out this template and not simply demand new features or send in complaints? Thanks! -->
 <!-- Also please search the existing issues (both open and closed) to see if your report might be duplicate -->
-<!-- Please don't file an issue when you have a support question, send support questions to the mailinglist or ask them on IRC (https://www.powerdns.com/opensource.html) -->
-<!-- Before filing your ticket, please read our 'out in the open' support policy at https://blog.powerdns.com/2016/01/18/open-source-support-out-in-the-open/ -->
+- [ ] This is not a support question, I have read [about opensource](https://www.powerdns.com/opensource.html) and will send support questions to the mailinglist or ask them on IRC.
+- [ ] I have read the ['out in the open' support policy](https://blog.powerdns.com/2016/01/18/open-source-support-out-in-the-open/) and am and will comply with it.
```
[the remark this answers is not in the document]
Comment (jsoref) [reply .github/ISSUE_TEMPLATE/bug_report.md:13]: Yeah, I tripped on some English...
```suggestion
- [ ] I have read the ['out in the open' support policy](https://blog.powerdns.com/2016/01/18/open-source-support-out-in-the-open/) and am complying with it now and will continue to comply with it.
```

[the remark this answers is not in the document]
Comment (Habbie) [reply .github/ISSUE_TEMPLATE/bug_report.md:13]: I prefer a simple "have read and understood"

[the remark this answers is not in the document]
Comment (Habbie) [reply .github/ISSUE_TEMPLATE/bug_report.md:13]:
```suggestion
- [ ] I have read and understood the ['out in the open' support policy](https://blog.powerdns.com/2016/01/18/open-source-support-out-in-the-open/)
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -9,8 +9,8 @@ assignees: ''
 
 <!-- Hi! Thanks for filing an issue. It will be read with care by human beings. Can we ask you to please fill out this template and not simply demand new features or send in complaints? Thanks! -->
 <!-- Also please search the existing issues (both open and closed) to see if your report might be duplicate -->
-<!-- Please don't file an issue when you have a support question, send support questions to the mailinglist or ask them on IRC (https://www.powerdns.com/opensource.html) -->
-<!-- Before filing your ticket, please read our 'out in the open' support policy at https://blog.powerdns.com/2016/01/18/open-source-support-out-in-the-open/ -->
+- [ ] This is not a support question, I have read [about opensource](https://www.powerdns.com/opensource.html) and will send support questions to the mailinglist or ask them on IRC.
```
Comment (Habbie) [inline .github/ISSUE_TEMPLATE/bug_report.md:12]:
```suggestion
- [ ] This is not a support question, I have read [about opensource](https://www.powerdns.com/opensource.html) and will send support questions to the IRC, [Github Discussions](https://github.com/PowerDNS/pdns/pull/13086/files) or the mailing list.
```

**Comment by Habbie** [comment]
I have applied some changes. Would like a second pair of eyes (from anybody) in case I did something stupid.

**Review by horazont — APPROVED** [review]

**Review by dwfreed — APPROVED** [review]

**Pull request merged**